polling_stations/apps/dashboard/templates/dashboard/pollingstation_detail.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load humanize %}
+
+{% block page_title %}{{ pollingstation.address }}{% endblock page_title %}
+
+{% block content %}
+<h1>{{ pollingstation.address }}</h1>
+
+    <p><a href="{% url "dashboard:council_detail" pollingstation.council_id %}">{{ pollingstation.council }}</a></p>
```

Comment:
This is a bit of an odd page. Is it missing something?

Reply:
It's mostly a placeholder, while I work out what might be useful to put on it.

Reply:
TODO: something useful here ;)